VEHICLE/SetTransformToSubmarineUsesAlternateInput.md

```diff
@@ -0,0 +1,89 @@
+---
+ns: VEHICLE
+aliases: ["0x41B9FB92EDED32A6", "_SET_UNK_BOOL_0x102_FOR_SUBMARINE_VEHICLE_TASK"]
+---
+## SET_TRANSFORM_TO_SUBMARINE_USES_ALTERNATE_INPUT
+
+```c
+// 0x41B9FB92EDED32A6
+void SET_TRANSFORM_TO_SUBMARINE_USES_ALTERNATE_INPUT(Vehicle vehicle, bool toggle);
+```
+
+Changes the key used to transform a vehicle into submarine mode. When set to true, the transformation key switches from the default raise/lower convertible roof key (usually 'H') to the special vehicle transformation key (usually 'X').
```

Comment:
Coule be replace by something more simple like: `Configure the submarine car to initiate the transform animation using the X/A button.`